### ISSUE TITLE
local gradient or integrated gradients of states [Feature Request] request title

### ISSUE THREAD

**Issue by 1900360** [DLR-RM/stable-baselines3#1365]
### 🚀 Feature

Is it possible to calculate the local gradient or integrated gradients of states in tianshou platform, and find out what states are important or not.

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo

**Comment by araffin** [comment]
Hello,
could you elaborate a bit more?

**Comment by araffin** [comment]
Duplicate of https://github.com/thu-ml/tianshou/issues/821

**Comment by 1900360** [comment]
I suppose SAC algorithm has one actor network and two critic network, now I want to rank the DRL states importance by calculate integrated gradients of each states to sork the states. so I wound if there is any possible to calculate like this:

```
self.actor.eval()      #actor network
obs = torch.tensor(batch.obs, requires_grad=True)   # observations   and open the obs_grad
baseline = torch.zeros((obs).shape, 3)    #don't know baseline meaning
attributions, delta = self.integrated_gradients.attribute(obs, target=0, return_convergence_delta=True)    #calculate integrated gradients of each states, but didn't work
obs.requires_grad_(False)        #close the obs_grad
```
